tests/test_wsgi_servers.py

```diff
@@ -29,6 +31,8 @@ def _gunicorn_args(host, port):
         'gunicorn',
         '--access-logfile', '-',
         '--bind', '{}:{}'.format(host, port),
+        '--graceful-timeout', '10',
+        '--timeout', '10',
```

Comment:
shouldn't there also use `_STARTUP_TIMEOUT`?

Reply:
Umm.. no, not really. If anything, `--graceful-timeout` could be somehow derived from `_SHUTDOWN_TIMEOUT` so that `--graceful-timeout` < `_SHUTDOWN_TIMEOUT`.

`--timeout` I added just in case, it should actually constitute a test failure if that happens, but we would a see a log item in Gunicorn's `stderr`, so it would be easier to reason about what has happened.

Reply:
I could maybe set `--graceful-timeout` to `_SHUTDOWN_TIMEOUT // 2`, but not sure if it would make anything clearer?

Reply:
I think the issue is that they have the same value, 10, so they same related. Maybe changing one of the makes sense?

Reply:
I can add `NOTE`s...

Reply:
I've hopefully clean this up now. Let me know what you think.

Reply:
ok now. btw above instead of issue I should have said that I made that remark because both had the same value